### DISCUSSION TITLE
یه ایده س دیگه

### DISCUSSION THREAD

**Discussion by tsalehm** [GFW-knocker/gfw_resist_tls_proxy#100] [Ideas]
یه روشی دیدم blindtls روش جالبی بود مطلب در موردش هست توی اینترنت. خلاصه کارش این بود که میومد داخل خود سرور وی پی ان یه کانکشن با سرور برقرار می کرد و با استفاده از TLS session resumption اون سیزن رو ادامه میداد. خب قطعا بستنش برای gfw آب خوردنه کلا هر پکتی با این درخواست رو دراپ می کنه

ولی یه ایده زد سرم با این فرگمنت کردن ترکیب بشه که نتونه بفهمه در اصل داریم درخواست از سرگیری اتصال میدیم. اونوقت اگه تونستیم وصل شیم شاید بشه یه ارتباط فیک تی ال اس با سرور برقرار کرد. که به اشتباه بیفته فکر کنه داریم به وایت لیست مقدسش وصل میشیم. الان یسری سرور ها با ایپی تمیز تقریبا همین رو پیاده می کنن بدون سی دی ان هاست فیک میزنن مثلا اسنپ و فلان و بمان و وصل میشن
 البته مطمعن نیستم کار بده یا نه ولی خب گفتم ایده رو به اشتراک بزارم

**Comment by GFW-knocker** [comment]
مرسی اتفاقا ایده بسیار بسیار خوبیه
بخصوص اگر بشه sni رو عوض کرد بعد resume

**Comment by 0x05E1** [comment]
یکم سعی کردم راش بندازم. برای کلادفلر که کلا کار نمی کنه به دلیل ممنوع بودن domain fronting
بعد اومدم برای gcore راش بندازم که اصلا حتی فرگمنت هم روش جواب نداد
خلاصه خیلی امیدی به این نیست

**Reply by tsalehm** [reply]
متوجه نشدم! ربطی به دامین فرانتینگ که نداره این روش ؟؟

**Reply by 0x05E1** [reply]
شایدم نداره نمی دونم. ولی به هر حال sni و هدر Host در blindtls یکی نیستن. ممکنه به همین دلیل کلادفلر بلاکش می کنه.
